src/actor.js

```diff
@@ -764,3 +764,45 @@ export const getApifyProxyUrl = (options = {}) => {
 
     return `http://${username}:${password}@${hostname}:${port}`;
 };
+
+/**
+ *
+ * Creates an ad-hoc webhook for the current actor run, which lets you receive a notification when the actor run finished or failed.
+ * For more information about Apify actor webhooks, please see the <a href="https://apify.com/docs/webhook" target="_blank">documentation</a>.
+ *
+ * Note that webhooks are only supported for actors running on the Apify platform.
+ * In local environment, the function will print a warning and have no effect.
+ *
+ * @param options.eventTypes {String[]} - Array of event types, which you can set for actor run, see
+ * the <a href="https://apify.com/docs/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.
+ * @param options.requestUrl {String} - URL which will be requested using HTTP POST request, when actor run will be in specific event type.
+ *
+ * @return {Promise<Object|undefined>}
+ *
+ * @memberof module:Apify
+ * @function
+ * @name webhook
+ */
+export const webhook = async ({ eventTypes, requestUrl }) => {
```

Comment:
One more thing. I know that in db and in client we use `requestUrl`, but IMHO it would be nicer to use just `url` here. This is what the people will see, and simplicity is the king.

Reply:
I'm for requestUrl to have it same everywhere.
Otherwise, we should change it in client, API, app to just url.

Reply:
We have requestUrl in doc https://apify.com/docs/webhooks#adhoc, which I link from SDK doc.

Reply:
Alright, let's go with `requestUrl` then. We can always deprecated it later :)

Reply:
I'm thinking again about `Apify.webhook` and maybe it's not the best idea. For example, if we decide to add function to remove webhook later, we'll have a problem. So I'd go with `Apify.addWebhook`. Sorry for the confusion.

Reply:
I can change it, but we everywhere creating, why are we adding in this case?